mcs/class/System.Windows.Forms/Test/System.Windows.Forms/DataBindingTests.cs

```diff
@@ -219,7 +219,7 @@ public void TestDeleteRowBeforeCurrent ()
 
 			Assert.AreEqual (0, cm.Position);
 
-			Console.WriteLine (event_log);
+			//Console.WriteLine (event_log);
```

Comment:
it'd be nice to use `DebugWriteLine` for these too

Reply:
At first I thought about doing that, but there are asserts just on the next line for each of these, so the value is verified. And in case the assert fails the value is printed anyway. Maybe it would be better to just remove it altogether?

Reply:
Yeah let's remove it in that case.